README.md

```diff
@@ -3,55 +3,124 @@ Karhunen Loève decomposed Gaussian processes with forward variable
 ```
-model = FoKLRoutines.FoKL(phis, relats_in, a, b, atau, btau, tolerance, draws, gimmie, way3, threshav, threshstda, threshstdb, aic)
+model.btau  = 1000
+model.draws = 2000
+model.way3  = 1
 ```
-- The definition of each of these hypers can be found within the function documentation.
+The above is useful for performing sweeps through hyperparameters without needing to initialize a new model (i.e., a new Python class) for each new combination of hyperparameters.
 
-With the model defined the training can begin by calling the fit function:
+The default hyperparameters and their keywords are as follows:
 ```
-model.fit( Normalized Training Inputs, Training Data)
+phis       = getKernels.sp500()
+relats_in  = []
+a          = 4
+b          = f(a, data)
+atau       = 4
+btau       = f(atau, data)
+tolerance  = 3
+draws      = 1000
+gimmie     = False
+way3       = False
+threshav   = 0.05
+threshstda = 0.5
+threshstdb = 2
+aic        = False
 ```
+A description of each hyperparameter is listed in the function documentation.
 
-The console will display the index and bic of the model being built in real time.
-Once completed the model can be validated with the coverage3 function:
+## Training
+Call the 'fit' function to train the FoKL model on all of 'data'.
+```
+betas, mtx, evs = model.fit(inputs, data)
+```
+Or, define the keyword 'train' as the percentage of 'data' to use for training.
+```
+betas, mtx, evs = model.fit(inputs, data, train=0.8)
+```
+The console will display the index and bic of the model being built in real time. Once completed, the model can be validated with the 'coverage3' function:
+```
+meens, bounds, rmse = model.converage3(model.inputs, model.data)
+```
+If validating visually, then a sorted plot tends to be most insightful. Explore the function documentation for more information, but to get started:
 ```
-model.converage3(Normalized Test Inputs, Test Data, draws, plots)
+model.converage3(model.inputs, model.data, plot='sorted', bounds=1, legend=1)
 ```
-Cover
+Also insightful is the RMSE of the model's fit, which is the third positional output of 'coverage3'.
 
-## Integration
-FoKL can be used to model state derivatives and thus contains an integration method of these states using an RK4. Due to each state being modeled independently the same functionality cannot be used. The model.fits outputs should be returned to your workspace via:
+As a side note, the following attributes were added to your FoKL class 'model' after calling 'fit' which may be useful during user post-processing:
+```
+model.inputs         == all normalized inputs w/o outliers (i.e., model.traininputs plus model.testinputs)
+model.data           == all data w/o outliers (i.e., model.traindata plus model.testdata)
+
+model.betas          == betas
+model.mtx            == mtx
+model.evs            == evs
+
+model.rawinputs      == all normalized inputs w/ outliers == user's 'inputs' but normalized and formatted
```

Comment:
Line 79 - if you mention outliers I think you should mention how you define outliers and the outlier methods in here